Translation corrections (#510)
07th-mod/higurashi-console-arcs: Update/kage_tips_135.txt

//rでも、さすがにこのステキな自然の中だと他にすることがなくて、ゆっくり休むしかないと思いますから。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "でも、さすがにこのステキな自然の中だと他にすることがなくて、ゆっくり休むしかないと思いますから。",

> 			NULL, "But now that she's all the way out in the middle of nothing but nature, she finally has no choice but to rest. That's what I believe.", Line_Normal);

Reviewer: ```suggestion
			NULL, "But now that out here in the middle of this lovely nature, she finally has no choice but to rest. That's what I believe.", Line_Normal);
```

//r警視総監…は、さすがに無理でもそれに近いくらいの高級官僚になって、世の中の平和のためにご活躍されることを私と私の家族、そしてお姉ちゃんと一緒に祈っております。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "警視総監…は、さすがに無理でもそれに近いくらいの高級官僚になって、世の中の平和のためにご活躍されることを私と私の家族、そしてお姉ちゃんと一緒に祈っております。",

> 			NULL, "Me, my family, and I pray that you become Superindendent General of the Metropolitan Police... well, perhaps that's too much of a stretch, but we hope you reach a position close to it and continue working for world peace.", Line_Normal);

Reviewer: ```suggestion
			NULL, "My family and I pray that you become Superintendent General of the Metropolitan Police... well, perhaps that's too much of a stretch, but we hope you reach a position close to it and continue working for world peace.", Line_Normal);
```
